pyresample/geometry.py

```diff
@@ -1258,6 +1258,9 @@ def to_cartopy_crs(self):
                   self.area_extent[2],
                   self.area_extent[1],
                   self.area_extent[3])
+        if Proj(**self.proj_dict).is_latlong():
```

Comment:
I think this is fine for now. Could you change this to not have the `**`. In my other PR I try to let `Proj` decide what to do with the information it is given (whether dict or str or something else). Not a huge deal, but `**` also isn't needed.

I suppose in the long run (pyresample 2.0) we will either use the `CRS` object directly and/or use the `Transformer` functionality used in pyproj 2.0+ instead of `Proj`.

Reply:
Good point, done!